### ISSUE TITLE
Riotclientservice.exe started but no grahpcis + stopped after 1-2mins

### ISSUE THREAD

**Issue by jakejakeho** [nhubaotruong/league-of-legends-linux-garena-script#40]
Started initial process 8288 from gamemoderun /home/jake/.local/share/lutris/runners/wine/lutris-ge-lol-7.0-1-x86_64/bin/wine /home/jake/Games/garena/drive_c/Garena/Games/32775/Riot Client/RiotClientServices.exe --garena-plus-token=myotken123456== --region=TW -- --locale=zh_TW --landing-token=fdeefb4606b9bf4669878661b1b4a4e7d07eb6cf10def7ed9f296d7fec9793b7 --servers.chat.chat_host=tw1.chat.si.riotgames.com --servers.lcds.lcds_host=prodtw1.lol.garenanow.com --servers.lcds.login_queue_url=https://lqtw1.lol.garenanow.com/login-queue/rest/queues/lol --region=TW --parent-client=GarenaPC
Start monitoring process.
gamemodeauto: 
esync: up and running.
Monitored process exited.
Initial process has exited (return code: 0)
Exit with return code 0

Riotclientservice.exe started but no graphics + stopped after 1-2mins

**Comment by nhubaotruong** [comment]
can you please show me your Garena and LOL lutris config ? Also what gpu are you using ?

**Comment by jakejakeho** [comment]
![image](https://user-images.githubusercontent.com/34710113/174478466-65efd39e-29ac-4433-9aac-ac84895ed5a7.png)
![image](https://user-images.githubusercontent.com/34710113/174478476-009349ef-9974-443d-bc0b-002b1f7977b9.png)

**Comment by nhubaotruong** [comment]
Previously a guy report to me he had the same issue, but he's using nvidia gpu, what are you using ?

**Comment by jakejakeho** [comment]
hi @nhubaotruong , thanks for your reply.
I am using AMD 5700

**Comment by jakejakeho** [comment]
![image](https://user-images.githubusercontent.com/34710113/174481940-ae48f095-3d13-40e2-b4f0-0401c0b02b0c.png)

**Comment by khewweifeng** [comment]
I use Mesa Intel® HD Graphics 620 (KBL GT2) and I also facing same issue like @jakejakeho 

Below are the log for garena (wine)

Started initial process 3540 from /home/acer/.local/share/lutris/runners/wine/lutris-ge-lol-7.0-2-x86_64/bin/wine /home/acer/Games/garena/drive_c/Program Files (x86)/Garena/Garena/Garena.exe
Start monitoring process.
esync: up and running.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

[0627/213703:ERROR:network_change_notifier_win.cc(153)] WSALookupServiceBegin failed with: 8

**Comment by khewweifeng** [comment]
> I use Mesa Intel® HD Graphics 620 (KBL GT2) and I also facing same issue like @jakejakeho
> 
> Below are the log for garena (wine)
> 
> Started initial process 3540 from /home/acer/.local/share/lutris/runners/wine/lutris-ge-lol-7.0-2-x86_64/bin/wine /home/acer/Games/garena/drive_c/Program Files (x86)/Garena/Garena/Garena.exe Start monitoring process. esync: up and running. MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0
> 
> MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0
> 
> [0627/213703:ERROR:network_change_notifier_win.cc(153)] WSALookupServiceBegin failed with: 8
> 
> ![Screenshot from 2022-06-27 21-43-12](https://user-images.githubusercontent.com/54496743/175956327-8c33f189-9101-4318-8549-ca397217b5f6.png) ![Screenshot from 2022-06-27 21-43-32](https://user-images.githubusercontent.com/54496743/175956332-129acfbb-05dc-4fcf-8b26-fd56ef4691e9.png)

![Screenshot_20220629_101958](https://user-images.githubusercontent.com/54496743/176343539-c17596d0-cfed-4f8d-8b57-2bab215ccf87.png)

**Comment by khewweifeng** [comment]
> > I use Mesa Intel® HD Graphics 620 (KBL GT2) and I also facing same issue like @jakejakeho
> > Below are the log for garena (wine)
> > Started initial process 3540 from /home/acer/.local/share/lutris/runners/wine/lutris-ge-lol-7.0-2-x86_64/bin/wine /home/acer/Games/garena/drive_c/Program Files (x86)/Garena/Garena/Garena.exe Start monitoring process. esync: up and running. MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0
> > MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0
> > [0627/213703:ERROR:network_change_notifier_win.cc(153)] WSALookupServiceBegin failed with: 8
> > ![Screenshot from 2022-06-27 21-43-12](https://user-images.githubusercontent.com/54496743/175956327-8c33f189-9101-4318-8549-ca397217b5f6.png) ![Screenshot from 2022-06-27 21-43-32](https://user-images.githubusercontent.com/54496743/175956332-129acfbb-05dc-4fcf-8b26-fd56ef4691e9.png)
> 
> ![Screenshot_20220629_101958](https://user-images.githubusercontent.com/54496743/176343539-c17596d0-cfed-4f8d-8b57-2bab215ccf87.png)

![Screenshot_20220629_111044](https://user-images.githubusercontent.com/54496743/176343610-5c10d468-6883-4edd-a83a-c325093181e4.png)

**Comment by nhubaotruong** [comment]
For me the game is running when using wine lutris lol 7.0.1 and the injector startup script, have you try it ?

**Comment by khewweifeng** [comment]
> For me the game is running when using wine lutris lol 7.0.1 and the injector startup script, have you try it ?

Hi @nhubaotruong , do we need to remove the 7.0.2 ?

**Comment by khewweifeng** [comment]
> > For me the game is running when using wine lutris lol 7.0.1 and the injector startup script, have you try it ?
> 
> Hi @nhubaotruong , do you mean run lol with lutris lol 7.0.1 ?
> 
> how about the garena ? because my garena will have flickering when I use lutris lol 7.0.1

![image](https://user-images.githubusercontent.com/54496743/176599818-99447a71-def9-4f3b-b430-5368b986c8b1.png)

result was not responding , if I choose to wait ... it will pop up again the same message as above screenshot

**Comment by khewweifeng** [comment]
> 

[log garena wine lutris 7.0.2.docx](https://github.com/nhubaotruong/league-of-legends-linux-garena-script/files/9016820/log.garena.wine.lutris.7.0.2.docx)

**Comment by khewweifeng** [comment]
> > > For me the game is running when using wine lutris lol 7.0.1 and the injector startup script, have you try it ?
> > 
> > 
> > Hi @nhubaotruong , do you mean run lol with lutris lol 7.0.1 ?
> > how about the garena ? because my garena will have flickering when I use lutris lol 7.0.1
> 
> ![image](https://user-images.githubusercontent.com/54496743/176599818-99447a71-def9-4f3b-b430-5368b986c8b1.png)
> 
> I tried with lol.py (with the injector startup script) and it did prompted me to enter root password
> 
> after that, result was not responding , if I choose to wait ... it will pop up again the same message as above screenshot, force quit will ended the processes.

[log lol wine lutris 7.0.2.docx](https://github.com/nhubaotruong/league-of-legends-linux-garena-script/files/9016822/log.lol.wine.lutris.7.0.2.docx)
[log lol wine lutris 7.0.1.docx](https://github.com/nhubaotruong/league-of-legends-linux-garena-script/files/9016824/log.lol.wine.lutris.7.0.1.docx)
[log garena wine lutris 7.0.2.docx](https://github.com/nhubaotruong/league-of-legends-linux-garena-script/files/9016825/log.garena.wine.lutris.7.0.2.docx)
[log garena wine lutris 7.0.2.docx](https://github.com/nhubaotruong/league-of-legends-linux-garena-script/files/9016827/log.garena.wine.lutris.7.0.2.docx)
[log lol wine lutris 7.0.1.docx](https://github.com/nhubaotruong/league-of-legends-linux-garena-script/files/9016831/log.lol.wine.lutris.7.0.1.docx)

**Comment by khewweifeng** [comment]
Running /home/acer/.local/share/lutris/runners/wine/lutris-ge-lol-7.0-1-x86_64/bin/wine /home/acer/Games/garena/drive_c/Garena/Games/32771/Riot Client/RiotClientServices.exe --garena-plus-token=WA4vca0GE2kMFaaWtLdNl6HwssxdmoM8ZPsGVmSBBh5OOzKuyOQUhm+byoPrXmzi5WcyqYiTc/AWR5HWbnqHmnelcvkoi2Vowl4+d+0BkGJEJxIPLVJA06koaqeG0iPMpeYj12s0/yBBL9g4Mwh2N6c8xtccoSpFos7gx+Qnh0MR5tgBXRSmqzUFD7wFfHKIeVw/PcHNZEBy4hdPGz/53Vws4gPnxYrYJNd4TcBnXsjuBYMZJq2CWJ/trinGQwudRyDY7PiCHvEN2mDiqBEReKFQgHh45vqnlMoc1n2LKimu0eNkqyedkygCUIuqPY8dJ/Uriv9z1Qf/Wo2E5dRQhQ== --region=SG -- --locale=en_SG --landing-token=0cbb814d23a822978f368859285ca623c141f3ae61fa4048458b3cbdab5d5f38 --servers.chat.chat_host=sg1.chat.si.riotgames.com --servers.lcds.lcds_host=prod.lol.garenanow.com --servers.lcds.login_queue_url=https://lq.lol.garenanow.com/login-queue/rest/queues/lol --region=SG --parent-client=GarenaPC
Initial process has started with pid 29047
Start monitoring process.
wineserver: using server-side synchronization.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

wine: configuration in L"/home/acer/Games/league-of-legends" has been updated.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

**Comment by khewweifeng** [comment]
Running /home/acer/.local/share/lutris/runners/wine/lutris-ge-lol-7.0-2-x86_64/bin/wine /home/acer/Games/garena/drive_c/Program Files (x86)/Garena/Garena/Garena.exe
Initial process has started with pid 27116
Start monitoring process.
esync: up and running.
ATTENTION: default value of option mesa_glthread overridden by environment.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

ATTENTION: default value of option mesa_glthread overridden by environment.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

wine: Call from 7BC2F920 to unimplemented function dxgi.dll.DXGID3D10CreateDevice, aborting
ATTENTION: default value of option mesa_glthread overridden by environment.
wine: Call from 7BC2F920 to unimplemented function dxgi.dll.DXGID3D10CreateDevice, aborting
ATTENTION: default value of option mesa_glthread overridden by environment.
wine: Call from 7BC2F920 to unimplemented function dxgi.dll.DXGID3D10CreateDevice, aborting
[0701/133549:ERROR:network_change_notifier_win.cc(153)] WSALookupServiceBegin failed with: 8
ATTENTION: default value of option mesa_glthread overridden by environment.
MESA-INTEL: warning: Performance support disabled, consider sysctl dev.i915.perf_stream_paranoid=0

wine: Call from 7BC2F920 to unimplemented function dxgi.dll.DXGID3D10CreateDevice, aborting
wine: Unimplemented function dxgi.dll.DXGID3D10CreateDevice called at address 7BC2F920 (thread 029c), starting debugger...
Unhandled exception: unimplemented function dxgi.dll.DXGID3D10CreateDevice called in segmented 32-bit code (0107:7bc2f920).
Assertion failed: curr_mode == stm_32bit, file /home/ubuntu/buildbot/runners/wine/wine-src/dlls/dbghelp/cpu_i386.c, line 278

**Comment by khewweifeng** [comment]
I stuck at this part 

![image](https://user-images.githubusercontent.com/54496743/176832005-08d601c6-c63a-48ac-ac4d-b7600d698187.png)

I downloaded your file and placed in same folder , use lol.py 

I use mx linux kde (debian 11 based)

**Comment by nhubaotruong** [comment]
Glorious Eggroll updated the `lutris-ge-lol-7.0-4-x86_64` runner, now it works ootb, just change Garena to using it (with DXVK enabled), and press Play, no need to use any script

**Comment by khewweifeng** [comment]
> Glorious Eggroll updated the `lutris-ge-lol-7.0-4-x86_64` runner, now it works ootb, just change Garena to using it (with DXVK enabled), and press Play, no need to use any script. Though you may have to navigate a black Garena screen

Do we still need to install riot LoL from lutris ?

**Comment by khewweifeng** [comment]
> Glorious Eggroll updated the `lutris-ge-lol-7.0-4-x86_64` runner, now it works ootb, just change Garena to using it (with DXVK enabled), and press Play, no need to use any script. Though you may have to navigate a black Garena screen

ya it works

**Comment by khewweifeng** [comment]
> Glorious Eggroll updated the `lutris-ge-lol-7.0-4-x86_64` runner, now it works ootb, just change Garena to using it (with DXVK enabled), and press Play, no need to use any script. Though you may have to navigate a black Garena screen

7.14 is working

**Comment by khewweifeng** [comment]
I tried with 7.0.2, let me try again and will feedback
